flume-s3-sink.iml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be committed. Please remove this file

Reply:
Its done now @amitsadafule